src/main/content/guides.html

```diff
@@ -45,8 +46,12 @@ <h2 id="guide_counter_title">All Open Liberty guides ({{ guides.size}})</h2>
                         <h3 class="guide_title">{{guide.title}}</h3>
                         <p class="guide_description">{{guide.description}}</p>
                     </div>
-                    <img class="duration_clock_icon" src="/img/guide_duration_clock_icon_small.svg" alt="Duration">
+                    <img class="duration_clock_icon" src="/img/guide_duration_clock_icon_small.svg" title="Duration" alt="Duration">
```

Comment:
We would need to consolidate the policy when it comes to adding tooltips via the title attribute in the site and the interactive guide. For now I would suggest not adding the title attribute on this particular file.